index.html

```diff
@@ -0,0 +1,50 @@
+<!-- @format -->
+
+<!DOCTYPE html>
+<html lang="en" class="html">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header class="upper">
+      <div class="toolbar">
+        <p id="name">Yilikal Teshome</p>
+        <nav class="drop">
+          <img id="icondrop" src="dropbox.png" alt="" />
+        </nav>
+      </div>
+    </header>
+    <section class="lower">
+      <div>
+        <h1>Hey There. I'm Yilikal</h1>

```

Comment:
- [ ] Kindly align the spacing between the hx and header tag, check screenshot below ⬇️